packages/commonwealth/server/migrations/20230411172411-cosmos-v1-sandbox.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `
+        INSERT INTO "ChainNodes" (name, url, alt_wallet_url, balance_type, bech32) VALUES (
+          'cosmos sdk local',
+          'http://0.0.0.0:26657/',
```

Comment:
Any reason to use 0.0.0.0 here rather than localhost?

Reply:
We should probably add this to code requirements, so devs know to use localhost, and not 0.0.0.0. 

Kurtis fixed a bug a while back that was introduced by using the wrong one, I would expect that was fixing 0.0.0.0 to be localhost, and not the other way round? @kurtisassad

Reply:
This was very particular about 0.0.0.0. 

I could probably configure it differently, but I had a bunch of headaches until I settled on this.

http://localhost:26657/ will still work in the browser, but Cosmos seemed to be finicky about using this for endpoints.

Reply:
I had thought that there were some exceptionally contexts where we have to use 0.0.0.0 rather than the localhost, and in fact, I thought we had a bug last quarter that @kurtisassad  literally fixed by literally changing localhost to 0.0.0.0. Unless it was going the other direction. I'm not seeing it in the codebase when I looked for it. Rather, only localhost, which is our standard reference for that.

Reply:
Ok it sounds like we may not know the exact interface at build time? So we're sending it out over all network interfaces? (which is what 0.0.0.0 does) 

@mhagel If you have any additional context for how we are using interfaces other than the loopback address, can we memorialize here? And just so it's clear (since Jake had mentioned localhost) the localhost loopback address is of course 127.0.0.1. So the question here is really why it needs to be sent out over all network interfaces, and not just loopback.

Reply:
Planning to move this to a live node for ease of testing. Heroku wasn't a good fit (we need ports 1317 and 26657 exposed, and heroku seems to favor just standard web use cases). Probably AWS.

Reply:
Now live on AWS. Thanks @NakulManchanda !